wavefront_pyformance/runtime_metrics.py

```diff
@@ -103,12 +116,13 @@ def collect_processes(self):
     def collect_contextswitches(self):
         """Collect Context Switches."""
         ctx_switches = self.process.num_ctx_switches()
-        voluntary = ctx_switches[0]
-        involuntary = ctx_switches[1]
-        self.registry.gauge("ctxswitch.voluntary",
-                            tags=self.custom_tags).set_value(voluntary)
-        self.registry.gauge("ctxswitch.involuntary",
-                            tags=self.custom_tags).set_value(involuntary)
+        if ctx_switches:
+            voluntary = ctx_switches[0]
```

Comment:
Curious, why do you use indexes instead of accessing the attributes by name, e.g. `ctx_switches.voluntary`

Reply:
@ustinov Started with indices and it was left like that. Changed it to names. Thanks.